### PR TITLE
feat(ui): implement new seed input form

### PR DESCRIPTION
**Pull request by rudolfs**
With this change:
  - we only allow adding and removing seeds one-by-one
  - allow the user to copy the seed to clipboard
  - perform validations
    - to make sure the seed is in a certain shape: `<peerId>@<host>:<port>`
    - to prevent adding duplicate seeds
  - ask the user to restart the app
    - the restart notification is made sticky and is always at the bottom of the notification stack
  - ~prompt the user to reload the peer so the configuration changes take effect~:
    - ~during the configuration reload we prevent the user from interacting with the UI by showing a busy cursor~
    - ~change the connectivity status indicator to "stopped" when the SSE channel is disconnected~

Closes: https://github.com/radicle-dev/radicle-upstream/issues/1029.

demo

### CODE REVIEW

**Review by cloudhead — APPROVED** [review]
Looks good :+1:

**Review by xla — CHANGES_REQUESTED** [review]
Code and UX look great, as discussed we can't guarantee proper reload so we should remove it for now.

There is one minor edge-case which is not a big deal, when an existing seed address is entered and without clearing the input the entry that was conflicting was deleted the validation still shows the same error.

File: ui/src/session.ts [hunk 1]
```diff
@@ -126,9 +129,87 @@ export const updateCoCo = (coco: CoCo): void =>
...
+
+  restartNotificationVisible = false;
+};
+
+const showRestartNotification = (): void => {
+  // The restart notification is sticky and will stay visible until the user
+  // clicks the "Restart" button, no need to show it multiple times.
+  if (restartNotificationVisible) return;
+
+  notification.error(
```
Comment (xla) [inline ui/src/session.ts:184]: I understand that we want to make sure the user doesn't miss this, but isn't it a bit of an abuse to show an error here? Can't we have something like a warning variant?

Comment (rudolfs) [reply ui/src/session.ts:184]: Good point. In the [style guide](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=586%3A0) we only have 2 levels so far.

Was actually thinking of renaming `error` in the code to `attention`.

Comment (rudolfs) [reply ui/src/session.ts:184]: Maybe @brandonhaslegs can let us know if we need a third variant?

Comment (brandonhaslegs) [reply ui/src/session.ts:160]: If we had persistent notifications (notifications that don't go away until you interact with them) then I would say keeping it black is fine. But this notification is **critical.** The user **must** restart to see results. Therefore, the user **must** see this notification. This is why I thought making it an error would be best. I don't see this as abuse, I see this as being helpful. If they miss this notification, they're confused, the app is "broken," they close the app and never come back.

Maybe I could settle for adding an "Attention" color. If we decide that's what we want to do, here's a design for it: https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=5863%3A47629

Comment (xla) [reply ui/src/session.ts:160]: Thanks for the context @brandonhaslegs - if I understand the code correctly the notification sticks around with this change, but maybe I'm off.

**Comment by rudolfs** [comment]
@xla addressed all your comments.

File: ui/src/session.ts (hunk 1 above)
Comment (rudolfs) [reply ui/src/session.ts:160]: @xla correct, we have persistent notifications now. Will change the notification type to `info` then, which will make the sticky notification black.

**Review by xla — APPROVED** [review]
🐎 🐑 🔱 💿

**Review by cloudhead — APPROVED** [review]

**Pull request merged**